packages/core/src/components/image-overlay/image-overlay.tsx

```diff
@@ -104,11 +104,17 @@ export class ImageOverlay implements ComponentInterface {
         {this.active && src && alt && (
           <div class="dimmer" ref={element => this.wrapperElement = element}>
             <div class="wrapper">
+              <div class="titel">
+                <slot name="titel" />
+              </div>
               <img src={src} alt={alt} />
               <button type="button" class="close" onClick={() => this.active = false}>
                 <dso-icon icon="times"></dso-icon>
                 <span>Sluiten</span>
               </button>
+              <div class="figuur-bijschrift">
```

Comment:
Als de afnemer nu geen bijschrift gebruikt rendered deze `div` alsnog. In viewer-grid.tsx staan constructies om lege slots ook niet te renderen.

Reply:
Hidden wanneer de slot niet wordt gebruikt.